src/hooks/index.ts

```diff
@@ -40,12 +38,7 @@ export function useEagerConnect() {
     if (wallet) {
       return SUPPORTED_WALLETS[wallet]?.connector
     }
-    if (window.xfi && window.xfi.ethereum) {
-      return xDefi
-    } else if (window.bitkeep && window.isBitKeep) {
-      return bitKeep
-    }
-    return window.ethereum
```

Comment:
I am interested to know the reason behind removal of this code. as i feel this was helpful in mobile device as you can see comment on 38th line number, lets make sure its not breaking mobile.